crates/ruff_dev/src/generate_options.rs

```diff
@@ -128,7 +128,11 @@ fn emit_field(output: &mut String, name: &str, field: &OptionField, parent_set:
     output.push_str(&format!(
         "**Example usage**:\n\n```toml\n[tool.ruff{}]\n{}\n```\n",
         if let Some(set_name) = parent_set.name() {
-            format!(".{set_name}")
+            if set_name == "format" {
+                String::from(".format")
+            } else {
+                format!(".lint.{set_name}")
+            }
```

Comment:
Unfortunately we may need to go through and add `.lint` to the appropriate settings, since some settings only exist at the top-level and aren't part of `.lint` (e.g., `.extend-exclude`).

Reply:
This scope is only for non-top-level settings.
Top-level names already excluded at [here](https://github.com/astral-sh/ruff/pull/8324/files#diff-f0bd91d69ddb50873c3b753b352c6e5e6dfbd791f662b04060cb6d70dbef60b0R130) and always replaced with empty string. did I understand wrong?

Reply:
Thanks -- you're right! Had to refresh my memory here.